Update readme (#2)
blindnet-io/sdk-node-server: README.md

<h1 align="center">
  blindnet devkit<br />
  Node.js SDK
</h1>

<p align=center><img src="https://user-images.githubusercontent.com/7578400/163277439-edd00509-1d1b-4565-a0d3-49057ebeb92a.png#gh-light-mode-only" height="80" /></p>
<p align=center><img src="https://user-images.githubusercontent.com/7578400/163549893-117bbd70-b81a-47fd-8e1f-844911e48d68.png#gh-dark-mode-only" height="80" /></p>

<p align="center">
  <strong>Node.js SDK for authentication tokens generation and user management</strong>
</p>

<p align="center">
  <a href="https://blindnet.dev"><strong>blindnet.dev</strong></a>
</p>

<p align="center">
  <a href="https://blindnet.dev/docs">Documentation</a>
  &nbsp;•&nbsp;
  <a href="https://github.com/blindnet-io/{project-short-name}/issues">Submit an Issue</a>
  &nbsp;•&nbsp;
  <a href="https://join.slack.com/t/blindnet/shared_invite/zt-1arqlhqt3-A8dPYXLbrnqz1ZKsz6ItOg">Online Chat</a>
  <br>
  <br>
</p>

## About

> This is the JavaScript [server-side SDK](https://docs.blindnet.io/docs/extra/glossary#server-side-sdk) component of [blindnet devkit][devkit] allowing you to:

Reviewer: ```suggestion
This is the JavaScript [server-side SDK](https://blindnet.dev/docs/glossary#server-side-sdk) component of [blindnet devkit][devkit] allowing you to:
```

> - Generate [authentication tokens](https://docs.blindnet.io/docs/extra/glossary#authentication-token) to allow users to access blindnet.

Reviewer: ```suggestion
- Generate [authentication tokens](https://blindnet.dev/docs/glossary#authentication-token) to allow users to access blindnet.
```

- Manage application users and data on blindnet.

## Get Started

:rocket: Check out our [Quick Start Guide](https://blindnet.dev/docs/quickstart) to get started in a snap.

## Installation

Use [npm][npm] or [yarn][yarn] to install the `@blindnet/sdk-javascript` package:

```bash

> npm install @blindnet/sdk-node

Reviewer: ```suggestion
npm install -S @blindnet/sdk-node
```
